plugins/include/message_const.inc

```diff
@@ -117,6 +117,55 @@ enum
 	ARG_ENTITY,         /* int */
 };
 
+/**
+ * Flags used in draw_status_icon()
+ */
+enum StatusIconFlags
+{
+	StatusIcon_Hide = 0,
+	StatusIcon_Show,
+	StatusIcon_Flash
+}
```

Comment:
Possibly missing semicolon, but compiler doesn't pay attention for it. Same below.